config/locales/en.yml

```diff
@@ -1256,23 +1256,25 @@ en:
         namespace_name_desc: Name, descending
         updated_at_asc: Oldest updated
         created_at_asc: Oldest created
-  pipeline_mailer:
-    complete_email:
-      subject: Pipeline Completed - %{id}
-      greeting: Hello %{name}
-      body_html: Your pipeline <strong>%{id}</strong> has completed successfully.
+  mailers:
+    automated_message: This email was sent from an automated system. Please do not reply to this email.
+    copy_paste: "Or copy and paste this link into your browser:"
+    greeting: Hello %{name},
```

Comment:
**suggestion:** Have you tried looking at using internationalized partials to do large blocks of text (see the first answer here: https://stackoverflow.com/questions/9172590/rails-strategies-for-internationalization-of-large-amounts-of-text-and-some-htm).  Just looking at this got me thinking there has to be a better way.  This would be great in so many areas if it works.  Let me know.

Reply:
I've added a template and refactored the current 3 emails to utilize the template with the `content_for`. Let me know what you think ead4a873f3f8f3ef5ad5dd4f674b2d0a3c897b7e